packages/hardhat-config/hardhat.config.js

```diff
@@ -0,0 +1,60 @@
+require("@nomiclabs/hardhat-etherscan")
+require("@nomiclabs/hardhat-ethers")
+require("@aragon/hardhat-aragon")
+
+const { homedir } = require("os");
+const { join } = require("path");
+
+// get the network url and account private key from ~/.aragon/network_key.json
+function getNetworkConfig(network) {
+  // default to mumbai
+  let url = "";
+  let accounts = [];
+
+  try {
+    const networkConfig = require(join(
+      homedir(),
+      `.aragon/${network}_key.json`
+    ));
+    url = networkConfig.rpc;
+    accounts = networkConfig.keys;
+  } catch (_) {}
+
+  return { url, accounts };
+}
+
+/**
+ * @type import('hardhat/config').HardhatUserConfig
+ */
+module.exports = {
+  solidity: {
+    version: "0.4.24",
```

Comment:
Since the version is hardcoded, and also `runs` count below in the settings, I might start to think that this can't be used for other projects such as `Govern` since compiler versions and runs  are different.

One way to solve this is to also pass all the parameters alongside `network`, but then, what's the point of having 1 file then if I have to pass everything almost ? 

Second way to solve this is we gotta somehow name the package that it only is usable for v1 aragon.

WDYT ?